### PR TITLE
Add paper sites

### PR DESCRIPTION
**Pull request by m2kar**
Add some common paper search and download website, which could link direct from lab.

### CODE REVIEW

**Comment by kslr** [comment]
This is not a valid file, Please read README

**Comment by m2kar** [comment]
> This is not a valid file, Please read README Rule

OK , I will modify and add to this PR. please keep this pr open.

**Comment by m2kar** [comment]
@kslr Could this changes meet the requirements in README.md ?

**Comment by m2kar** [comment]
How about now?

File: data/papers [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# 常见的查文献的网站
+# Common literature websites 
+
+# cn
+include:wanfang
+include:cnki
+include:cas
+
+# out of cn
+include:elsevier
```
Comment (kslr) [inline data/papers]: for cn, please add to geolocation-cn 
for "out of cn", please add to geolocation-!cn

**Comment by m2kar** [comment]
There may be mistaken. in some cases, researcher or student must visit these paper sites from the inner network of university, no matter these websites are Chinese local website or not. So I want to deploy a v2ray proxy inner my laboratory and visit these websites from home or other places out of the laboratory. 

In my solution, I make a collection of such paper sites and save to "papers" file and compile to `geosite.dat`. Then in my v2ray configure file, I add `geosite:papers` to the `route` field. I could achieve the desired function now. 

So I want to share this small improvement to others, there must be a lot of people who have such demands.

If this project does not allow such functional improvement, I will achieve in my own fork.

**Comment by kslr** [comment]
I recommend using Fork

**Comment by m2kar** [comment]
OK, Thank you 




------------------ 原始邮件 ------------------
发件人: "Github"<notifications@github.com>; 
发送时间: 2019年8月21日(星期三) 下午4:41
收件人: "v2ray/domain-list-community"<domain-list-community@noreply.github.com>; 
抄送: "芮志清"<ruizhiqing@foxmail.com>; "Author"<author@noreply.github.com>; 
主题: Re: [v2ray/domain-list-community] Add paper sites (#83)




I recommend using Fork
 
&mdash;

**Pull request merged**